catalog/transformer/catalog-transformer-geojson-input/src/main/java/ddf/catalog/transformer/input/geojson/GeoJsonInputTransformer.java

```diff
@@ -79,6 +83,8 @@
 
   private AttributeRegistry attributeRegistry;
 
+  private SortedServiceReferenceList inputTransformers;
```

Comment:
❓ Why the `SortedServiceReferenceList` instead of `SortedServiceList`?  You can avoid `FrameworkUtil` if you use `SortedServiceList`.

Reply:
I had some thought of being able to filter on service properties, but I think that would require more changes to the endpoint so I can switch.